src/components/products/Sidebar.jsx

```diff
@@ -0,0 +1,27 @@
+import "./Sidebar.scss"
+
+export default function Sidebar({ selected, handleSelect, categories }) {
+
+  const isSelected = (category) => selected.includes(category) ? "selected" : "not-selected"
+
+  return (
```

Comment:
Remember that the sidebar should also have a a button to clear all filters and when it is clicked all products should be shown.